packages/web/src/pages/settings-page/components/desktop/DeveloperApps/DeveloperAppListItem.module.css

```diff
@@ -4,19 +4,22 @@
   background-color: var(--harmony-n-25);
   border-radius: var(--harmony-unit-1);
   border: 1px solid var(--harmony-n-100);
-  gap: var(--harmony-unit-6);
-  padding: var(--harmony-unit-4) var(--harmony-unit-6);
+  padding: 0 var(--harmony-unit-6) 0 0;
+  overflow: hidden;
 }
 
-.listItemIndex {
+.listItemImage {
   flex-grow: 0;
-  font-size: var(--harmony-font-m);
-  font-weight: var(--harmony-font-bold);
-  color: var(--harmony-n-600);
+  height: 56px;
+  width: 56px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--harmony-n-100);
```

Comment:
Is there a semantic background color we can use here?
https://github.com/AudiusProject/audius-protocol/blob/eaba2e66bd9ad6cce73147061b4f91a9f4979da6/packages/harmony/src/foundations/color/semantic.css#L9